charts/iter8lib/templates/_task-readiness.tpl

```diff
@@ -0,0 +1,32 @@
+{{- define "task.readiness" -}}
```

Comment:
I want to propose a similar template, but simpler and easier to read, and as it happens, without any need for values in `iter8lib`.
***
```yaml 
# in a separate file
{{- define "task.ready.tn" }}
{{- if .Values.ready.timeout }}
timeout: {{ .Values.ready.timeout }}
{{- end }}
{{- if .Values.ready.namespace }}
namespace: {{ .Values.ready.namespace }}
{{- end }}
{{- if .Release.Namespace }}
namespace: {{ .Release.Namespace }}
{{- end }}
{{- end }}
```

```yaml
{{- define "task.ready" }}
{{- if .Values.ready.service }}
- task: k8s-object-ready
   with:
    name: {{ .Values.ready.service }}
    version: v1
    resource: services
{{- include "task.ready.tn" . | indent 4 }}
{{- end }}
{{- if .Values.ready.deploy }}
- task: k8s-object-ready
   with:
    name: {{ .Values.ready.deploy }}
    version: v1
    resource: deployments
    condition: Available
{{- include "task.ready.tn" . | indent 4 }}
{{- end }}
{{- end }}
```

Reply:
done